### ISSUE TITLE
Using -INI with -EDIT results  'Video graphics' in crash when closing dialog

### ISSUE THREAD

**Issue by MikedaSpike** [vpinball/vpinball#987]
**Describe the bug**
When launching Visual Pinball with -INI <inifilename> and -edit <tablefile>, go to "Video Graphics option" and close dialog, results that Visual Pinball crash.

Event viewer :

```
Log Name:      Application
Source:        Application Error
Date:          23-10-2023 06:09:19
Event ID:      1000
Task Category: Application Crashing Events
Level:         Error
Keywords:      
User:          PINBALL\Mike DA Spike
Computer:      Pinball
Description:
Faulting application name: VPinballX64.exe, version: 0.0.0.0, time stamp: 0x6535acb7
Faulting module name: VPinballX64.exe, version: 0.0.0.0, time stamp: 0x6535acb7
Exception code: 0xc0000005
Fault offset: 0x00000000000ca982
Faulting process id: 0x0x1254
Faulting application start time: 0x0x1DA0566ADEFCABC
Faulting application path: C:\Pinball\Visual Pinball VR\VPinballX64.exe
Faulting module path: C:\Pinball\Visual Pinball VR\VPinballX64.exe
Report Id: 3358238b-d086-4741-9a1a-86c1ca50b057
Faulting package full name: 
Faulting package-relative application ID: 
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" Guid="{a0e9b465-b939-57d7-b27d-95d8e925ff57}" />
    <EventID>1000</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>100</Task>
    <Opcode>0</Opcode>
    <Keywords>0x8000000000000000</Keywords>
    <TimeCreated SystemTime="2023-10-23T04:09:19.3247995Z" />
    <EventRecordID>77514</EventRecordID>
    <Correlation />
    <Execution ProcessID="15680" ThreadID="15844" />
    <Channel>Application</Channel>
    <Computer>Pinball</Computer>
    <Security UserID="S-1-5-21-3876027899-2469441806-1619404739-1001" />
  </System>
  <EventData>
    <Data Name="AppName">VPinballX64.exe</Data>
    <Data Name="AppVersion">0.0.0.0</Data>
    <Data Name="AppTimeStamp">6535acb7</Data>
    <Data Name="ModuleName">VPinballX64.exe</Data>
    <Data Name="ModuleVersion">0.0.0.0</Data>
    <Data Name="ModuleTimeStamp">6535acb7</Data>
    <Data Name="ExceptionCode">c0000005</Data>
    <Data Name="FaultingOffset">00000000000ca982</Data>
    <Data Name="ProcessId">0x1254</Data>
    <Data Name="ProcessCreationTime">0x1da0566adefcabc</Data>
    <Data Name="AppPath">C:\Pinball\Visual Pinball VR\VPinballX64.exe</Data>
    <Data Name="ModulePath">C:\Pinball\Visual Pinball VR\VPinballX64.exe</Data>
    <Data Name="IntegratorReportId">3358238b-d086-4741-9a1a-86c1ca50b057</Data>
    <Data Name="PackageFullName">
    </Data>
    <Data Name="PackageRelativeAppId">
    </Data>
  </EventData>
</Event>

```

**Expected behavior**
If just opening Editor with -INI <inifile> all is working fine

**Screenshots**
NA

**Crash dump**
(rename the files to .zip.001 , .zip002 etc)
[crash001.zip](https://github.com/vpinball/vpinball/files/13066065/crash001.zip)
[crash002.zip](https://github.com/vpinball/vpinball/files/13066064/crash002.zip)
[crash003.zip](https://github.com/vpinball/vpinball/files/13066069/crash003.zip)
[crash004.zip](https://github.com/vpinball/vpinball/files/13066068/crash004.zip)
[crash005.zip](https://github.com/vpinball/vpinball/files/13066066/crash005.zip)
[crash006.zip](https://github.com/vpinball/vpinball/files/13066067/crash006.zip)

**Versions & Settings**
 - VPX: 10.8 rev 1631
 - Windows version : W 11 22h2
 - GPU and video settings if applicable

**Comment by vbousquet** [comment]
I can not reproduce this one. Does it still happens to you?
Did you try to remove your ini settings file and see how it behaves now (I have changed many many things in settings, so you may have stumbled an unstable version, but it is supposed to be nearly finished now) ?

**Comment by MikedaSpike** [comment]
Thanks Vincent, i need to update to latest and check it. Hope to have an answer this week. 
Thanks for all your work for the 10.8 release. I (and many others) realy appreciate your hard work !

**Comment by MikedaSpike** [comment]
@vbousquet  
I tried it with 1633 64 today.
just `VPinballX64.exe -ini VPinballX64.ini -edit <Table name>`
go to video settings and click OK, crashed the exe
removed VPinballX64.ini and do the same, results the exe crashed.
removed VPinballX64.ini  and removed VPinballX.ini from %appdata%\vpinballx and launch the teh same, results in crash too
remove of  VPinballX64.ini  and  VPinballX.ini from %appdata%\vpinballx and launch `VPinballX64.exe', goto video options works fine and saves it
Launching `VPinballX64.exe -ini VPinballX64.ini 'works fine as well, but adding -edit with a table results in a crash again
I made a video of all the steps I mentioned above. maybe it is a bit unclear, but hope you get it
[https://youtu.be/UW_4urRnARY](url)

These are the latest ini files it created.
Vpinball64.ini is my custom one and VpinballX.ini is from appdata

[vpinballx64.txt](https://github.com/vpinball/vpinball/files/13201419/vpinballx64.txt)
[VPinballX.txt](https://github.com/vpinball/vpinball/files/13201420/VPinballX.txt)
